doc/060_forget.rst

```diff
@@ -195,6 +195,9 @@ The ``forget`` command accepts the following parameters:
     boundaries and not relative to when you run the ``forget`` command. Weeks
     are Monday 00:00 -> Sunday 23:59, days 00:00 to 23:59, hours :00 to :59, etc.
 
+.. note:: The ``--keep-tag`` option with an empty string '' will match untagged
```

Comment:
What about writing
```
``--keep-tag ""```
```
instead?

Reply:
Done.  I hope the changes are to your satisfaction.